package edu.pratik.proxy.exception;

public class TaskAlreadyAdded_Exception extends Exception {
    public TaskAlreadyAdded_Exception(String message, Throwable cause) {
        super(message, cause);
    }
}
